index.js

```diff
@@ -13,6 +13,7 @@ function expressPlugin (fastify, options, next) {
   fastify.use = use
```

Comment:
the TODO above here - should it be addressed?

Reply:
Yes.

Reply:
https://github.com/fastify/fastify-express/pull/1#discussion_r361797525 references a merged PR so I guess it was just forgotten?

Reply:
Yup, it got forgotten :D

Reply:
#8